weave-js/src/components/PagePanelComponents/util.ts

```diff
@@ -21,8 +21,17 @@ export type BranchPointType = {
   original_uri: string;
 };
 
+let IN_JUPYTER_CELL = false;
+
+export const detectInJupyterCell = () => {
```

Comment:
why have this hold state instead of just looking at location?

Reply:
because the location was being redirected when the page loads. I think Jamie fixed separately. Will post a follow-up